lib/main.dart

```diff
@@ -230,10 +230,14 @@ class _TenTenOneState extends State<TenTenOneApp> {
   }
 
   Future<void> _callSync() async {
-    final balance = await api.getBalance();
-    bitcoinBalance.update(Amount(balance.onChain.confirmed));
-    lightningBalance.update(Amount(balance.offChain));
-    FLog.trace(text: 'Successfully synced Bitcoin wallet');
+    try {
```

Comment:
🤔 run periodically is already wrapping that into a try catch - why is this needed?